src/animation/shine.js

```diff
@@ -27,9 +28,12 @@ const Shine = ({ children }) => {
       Animated.timing(animation, {
         toValue: END_VALUE,
         duration: DURATION,
+        useNativeDriver,
```

Comment:
I'm not sure `useNativeDriver` is useful right there since it's not applicable to `marginLeft`animation (https://facebook.github.io/react-native/docs/animations#caveats) 😊